dev-python/pipdeptree/pipdeptree-2.13.0.ebuild

```
@@ -0,0 +1,21 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=hatchling
+PYTHON_COMPAT=( pypy3 python3_{10..12} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="pipdeptree is a utility for displaying installed packages in a dependency tree."
```

No need for "... is a"
```suggestion
DESCRIPTION="Utility for displaying installed packages in a dependency tree"
```

---

Fixed.